CHANGELOG.md

```diff
@@ -12,6 +12,13 @@
 * Fixed a bug where **doc-review** command failed on existing templates.
 * Fixed a bug where **validate** command failed when the word demisto is in the repo README file.
 * Added support for adding test-playbooks to the zip file result in *create-content-artifacts* command for marketplacev2.
+* Refactored the **format** command on update *fromversion*:
+  - Fixed a bug when changing content item, the *fromversion* field should remain.
+  - Updated the general default *fromversion* and the default *fromversion* of new content items (e.g. `Lists, Jobs..`).
+  - Removed the interactive in *update_playbook*.
+  - Added interactive for all content types, asking user if to generate default *fromversion*.
+  - Updated *update_generic_yml/json* and their usages in the differences content types update.
```

Comment:
```suggestion
  - Updated *update_generic_yml/json* and their behavior when content items are modified.
```

unclear, if my suggestion makes sense please commit it, otherwise please elaborate a bit more

Reply:
You are right, an unnecessary remark.